spec/features/invitation_spec.rb

```diff
@@ -58,4 +59,40 @@ def last_email
   def email_count
     ActionMailer::Base.deliveries.count
   end
+
+  describe "insitution invite" do
+    context "new user" do
+      let!(:new_user) {
+        User.make(:invited_pending).tap do |user|
+          user.send("generate_invitation_token!")
+        end
+      }
+      let!(:invite) { PendingInstitutionInvite.make(invited_user_email: new_user.email, invited_by_user: user) }
+
+      it "onboards invitee" do
+        goto_page AcceptInvitiation, "query" => {invitation_token: new_user.raw_invitation_token, pending_institution_invite_id: invite.id} do |page|
+          expect(page).to have_content "Set your password"
+          expect(page.first_name.value).to eq new_user.first_name
+          expect(page.last_name.value).to eq new_user.last_name
+
+          page.password.set 'passw0rd'
+          page.password_confirmation.set 'passw0rd'
+          page.submit
+        end
+
+        expect_page InstitutionNewFromInvitePage do |page|
+          expect(page).to have_content("New Institution")
+          expect(page.name.value).to eq invite.institution_name
+          # TODO: Checking the radio button doesn't work for some reason
+          # expect(page).to have_checked_field("institution_kind_#{invite.institution_kind}")
+          # page.choose("institution_kind_manufacturer")
+
+          page.submit
+        end
+
+        # FIXME: This flash message probably shouldn't be there
+        expect(page).to have_content("You are already signed in")
```

Comment:
This flash message is probably a bug?

Reply:
Yeah, that sounds fishy. Maybe the test logs a user in already? And when we accept the invitation we try to log the user in _again_ and that prints that error?